### ISSUE TITLE
.json and training output are not the same

### ISSUE THREAD

**Issue by WilliamJudge94** [keras-team/autokeras#1413]
### Bug Reproduction
Read a completed trial.json file and compare it to it's training outputs. I am using 'val_loss' as the parameter to track during AutoKeras ImageClassifier Training. As you can see the accuracy and val_accuracy are taking values from epoch=1 while the loss and val_loss are taking their values from epoch=6. 

Purpose: I would like to track final val_loss and val_accuracy for each model created. 

<img width="524" alt="Screen Shot 2020-11-03 at 12 17 45 PM" src="https://user-images.githubusercontent.com/37912237/98025150-f3760a80-1dce-11eb-93de-8030e8ed90cd.png">

<img width="960" alt="Screen Shot 2020-11-03 at 12 11 29 PM" src="https://user-images.githubusercontent.com/37912237/98025156-f670fb00-1dce-11eb-92a1-69433fa448f9.png">

### Setup Details
Include the details about the versions of: (If blank they are Google Colab Standards)
 - OS type and version: 
 - Python:
 - autokeras: git clone https://github.com/keras-team/autokeras.git
 - keras-tuner: keras-tuner-1.0.2rc3
 - scikit-learn:
 - numpy: 
 - pandas:
 - tensorflow: 2.3.0

**Comment by haifeng-jin** [comment]
Thank you for the issue. We will inspect it.
